config.js

```diff
@@ -50,6 +51,7 @@ var environment = {
   awsRegion: process.env.AWS_REGION,
   sendgridApiKey: process.env.SENDGRID_API_KEY,
   sendgridFrom: process.env.SENDGRID_FROM,
+  gdalTranslateBin: process.env.GDAL_TRANSLATE_BIN,
```

Comment:
Skip this entirely and use `gdal_translate` as the command without specifying a path. If the `process-image.js` process is started with `gdal_translate` in its `PATH` (so `PATH` is set externally), it will work.

For it to work locally, ensure that `/Library/whatever/GDAL.framework/bin` is in `PATH` before starting the worker and you should be good to go.